src/ess/reduce/nexus/types.py

```diff
@@ -69,19 +69,13 @@ class TransmissionRun(Generic[ScatteringRunType]):
     """
 
 
-RunType = TypeVar(
-    'RunType',
-    BackgroundRun,
-    EmptyBeamRun,
-    SampleRun,
-    # Note that mypy does not seem to like this nesting, may need to find a workaround
-    TransmissionRun[SampleRun],
-    TransmissionRun[BackgroundRun],
-    VanadiumRun,
-)
+RunType = TypeVar('RunType')
```

Comment:
Do we need a deprecation period? Or do you plan to to a sync release of all downstream packages? For the former, we could first make the `run_type` and `monitor_type` args to `GenericNexusWorkflow` (and `GenericTofWorkflow`) mandatory. But maybe we are already using those in most downstream packages, so it might be a non-issue?

Reply:
AFAIK these types are only used with `GenericNexusWorkflow` or `GenericTofWorkflow` downstream. So making the arguments mandatory would break all the same packages as removing the constraints.
I think this will be a small issue because almost all code already passes `run_type` and `monitor_type`.